### DISCUSSION TITLE
ALARM 125 OPEN DTU / HMS_1000-2T

### DISCUSSION THREAD

**Discussion by vcarre83** [tbnobody/OpenDTU#2307] [Q&A]
Hello, I have encountered a problem since yesterday on one of my HMS_1000-2T. In fact I no longer have solar production and my open DTU logs indicate an error 125 (parameter error of
network setup
). 
I electrically shut off my inverter then turned it back on and the same error appeared.

Have you ever encountered this error?

![Capture d'écran 2024-09-25 092211](https://github.com/user-attachments/assets/d13128cf-75f7-4912-985c-d744254cdeeb)

**Comment by ms49434** [comment]
User manual is suggesting to check your grid profile. If you want to load or reload a grid profile, a Hoymiles installer account and a Hoymiles DTU is required. If the issue cannot be resolved, contact your dealer or Hoymiles customer service.
![image](https://github.com/user-attachments/assets/11417a5f-500e-42da-ba20-e136a75bbe4c)

**Comment by vcarre83** [comment]
Hello, thank you for the answer.

I have indeed seen this indication in the manual however I do not understand the manipulation to be carried out concerning the grid.
I have other HMS1000-2T inverters on my open DTU which work perfectly.

**Reply by ms49434** [reply]
Please confirm that the grid profile matches the requirement of your provider. To do that, double-check the grid profile of each inverter:
![image](https://github.com/user-attachments/assets/da62046d-c012-47ce-bd39-ada9374846fb)

**Comment by vcarre83** [comment]
The profile grid is not correct compared to my other HMS1000-2T inverters. It's strange everything was ok one evening then the next morning error 125.

![image](https://github.com/user-attachments/assets/ae025838-e53a-4622-a6f3-725863bb8982)

**Reply by ms49434** [reply]
The DIY path is to get a Hoymiles DTU and a Hoymiles installer account and apply the correct profile.

**Reply by stefan123t** [reply]
Could you @vcarre83 document the correct Grid Profile on one of your other HMS inverters. I understand that only one of your inverters somehow got this rather “short” profile update with v0.0.2 of US - NA_IEEE1547_240V. Basically this consists only of the Header and the CRC16 checksum there is no body 👀 in this Grid Profile 😱.

We already traced the basic Grid Profile update communication but it has not been implemented in OpenDTU code yet.

See here for an analysis of the Grid Profile Update command using the Logic Analyser Signal Trace of updating to a German DE_VDE4105_2018 Grid Profile by @nivadis.

https://github.com/tbnobody/OpenDTU/issues/900#issuecomment-2209546838

and earlier here

https://github.com/tbnobody/OpenDTU/issues/900#issuecomment-1899406249

**Comment by vcarre83** [comment]
OK thank you for your feedback. Could you tell me a compatible Hoymiles DTU ref? For the account it is a user account or a particular account to apply the correct profile? Finally, how could the profile have changed from the day before to the next day?

**Reply by ms49434** [reply]
You need a DTU-Pro-S or a DTU-Lite-S for HMS inverters.
The Hoymiles Gateway DTU-Pro S works fine for me. I can't say anything about the Hoymiles Gateway DTU-Lite-S.
On your mobile, install the S-Miles Installer app and follow the steps to create your installer/diy account in the Hoymiles cloud.

Source: https://www.hoymiles.com/products/microinverter/dtu/

**Reply by vcarre83** [reply]
Thank you for answer. I try this solution.

**Reply by wilko834** [reply]
Hello, @vcarre83 I have exactly the same problem. What was the solution? Thank you.

**Comment by vcarre83** [comment]
Hello, I re-injected the configuration via a Hoymiles usb/wifi gateway (https://www.amazon.fr/Hoymiles-DTU-wlite-contr%C3%B4le-alimentation-hm-800/dp/B0CD7WK8M1) to my HMS inverters. I then re-associated my inverters with Open DTU.
Everything returned to normal after the configuration injection.

**Reply by wilko834** [reply]
Thank you... Are you in France? In the 83? :)

**Comment by vcarre83** [comment]
Yes

**Reply by wilko834** [reply]
Par hasard vous avez pas un dtu qui traîne juste pour faire la synchro ? Je n'en ai pas, si ça peut m'éviter d'en acheter un juste pour ça...

**Comment by vcarre83** [comment]
Je l'ai revendu désolé

**Reply by wilko834** [reply]
Salut, voilà j'ai reçu mon dtu S... Tu as pu faire la mise à jour via l'application s miles installer ?
